lib/ansible/modules/network/fortios/fortios_ipv4_policy.py

```diff
@@ -0,0 +1,292 @@
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+ANSIBLE_METADATA = {
+    'status': ['preview'],
+    'supported_by': 'community',
+    'version': '1.0'
+}
+
+DOCUMENTATION = """
+---
+module: fortios_ipv4_policy
+version_added: "2.3"
+author: "Benjamin Jolivot (@bjolivot)"
+short_description: Manage fortios firewall IPv4 policy objects
+description:
+  - This module provides management of firewall IPv4 policies on FortiOS devices.
+extends_documentation_fragment: fortios
+options:
+  id:
+    description:
+      - Policy ID.
+    required: true
+  state:
+    description:
+      - Specifies if address need to be added or deleted.
+    choices: ['present', 'absent']
+    default: present
+  src_intf:
+    description:
+      - Specifies source interface name.
+    default: any
+  dst_intf:
+    description:
+      - Specifies destination interface name.
+    default: any
+  src_addr:
+    description:
+      - Specifies source address (or group) object name(s).
+    required: true
+  src_addr_negate:
+    description:
+      - Negate source address param.
+    default: false
+    choices: ["true", "false"]
+  dst_addr:
+    description:
+      - Specifies destination address (or group) object name(s).
+    required: true
+  dst_addr_negate:
+    description:
+      - Negate destination address param.
+    default: false
+    choices: ["true", "false"]
+  policy_action:
+    description:
+      - Specifies accept or deny action policy.
+    choices: ['accept', 'deny']
+    required: true
+    aliases: ['action']
+  service:
+    description:
+      - "Specifies policy service(s), could be a list (ex: ['MAIL','DNS'])."
+    required: true
+    aliases:
+      - services
+  service_negate:
+    description:
+      - Negate policy service(s) defined in service value.
+    default: false
+    choices: ["true", "false"]
+  schedule:
+    description:
+      - defines policy schedule.
+    default: 'always'
+  nat:
+    description:
+      - Enable or disable Nat.
+    default: false
+    choices: ["true", "false"]
+  fixedport:
+    description:
+      - Use fixed port for nat.
+    default: false
+    choices: ["true", "false"]
+  poolname:
+    description:
+      - Specifies NAT pool name.
+  av_profile:
+    description:
+      - Specifies Antivirus profile name.
+  webfilter_profile:
+    description:
+      - Specifies Webfilter profile name.
+  ips_sensor:
+    description:
+      - Specifies IPS Sensor profile name.
+  application_list:
+    description:
+      - Specifies Application Control name.
+  comment:
+    description:
+      - free text to describe policy.
+notes:
+  - This module requires pyFG library.
+"""
+
+EXAMPLES = """
+- name: Allow external DNS call
+  fortios_ipv4_policy:
+    host: 192.168.0.254
+    username: admin
+    password: password
+    id: 42
```

Comment:
1. I guess examples should use a string for the value of `id` parameter.
2. `name` should be used instead of `id` ([see](http://docs.ansible.com/ansible/dev_guide/developing_modules_best_practices.html)).